utils/manage_events.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing, please email
+# opensource@seagate.com or cortx-questions@seagate.com.
+
+
+import logging
+from typing import List
+
+
+from cortx.utils.message_bus import MessageBus, MessageConsumer
+from ha.core.event_manager.event_manager import EventManager
+
+
+class ManageEvents():
+    """
+    Typical use case of this class is as follows
+    subscribe(event list) - subscribe to events for listening, returns topic
+                            which can be used for listening
+    _listen(topic) - reading messages received on above topic
+    unsubscribe(event list) - unsubscribe as we dont want to receive these
+                              events anymore
+    get_message_type() - returns topic just in case you are not maintaining
+                         output of subscribe
+    """
+
+    event_manager = None
+
+    def __init__(self):
+        print('Inside __init__')
+        self.event_manager = EventManager.get_instance()
+
+    def subscribe(self, event_list: List[str]) -> str:
+        logging.info('Subscribing for events: (%s)', str(event_list)[1:-1])
+        kafka_topic_name = self.event_manager.subscribe('hare', event_list)
```

Comment:
`'hare'` could have been refactored into a constant or a field or a method invocation. Hardcoded strings (especially when they are repeated multiple times) is not a good thing.

Reply:
Done